docs/jobs-site.md

```diff
@@ -140,11 +140,25 @@ Once reviewed, approved and released, your requested files will be available to
 
 ## Publishing outputs
 
-You must seek NHS England approval for any publication or wider sharing of results, papers, presentations (e.g. submitting to a journal or a pre-print server, or uploading to any public facing website). 
+You must seek NHS England approval for any publication or wider sharing of results, papers, presentations (e.g. submitting to a journal or a pre-print server, or uploading to any public facing website).
 
 For instructions on how to request approval, please see [this section](https://www.opensafely.org/policies-for-researchers/#all-datasets-publication) of the researcher policy document. Following approval from NHSE, you should also create draft public outputs for review. To do this, navigate to the _release_ containing the outputs you would like to be published and click the `Publish` button.
 
 As part of publishing your outputs, you should also make the repository where your analysis code is written public. You can find instructions on how to do that [here](https://docs.opensafely.org/publishing-repo/).
 
 Once approved, your released outputs will be “published” and viewable from the published outputs in the `Published outputs` section of your `Releases`. This is accessible by everyone, even those without a login.
 
+
+## Updating project status
+When you start a new project on OpenSAFELY it will have the _Ongoing_ status, but as your work progresses you'll want to update this status.
+
+If you are a member of a project you will see an `Edit` button near the top of your project page:
+
+![Project edit button](./images/project-edit-button.jpg)
```

Comment:
thought: The text in `[ ]` is the alt tag. Should we be using more descriptive alt tags to improve accessibility? (It's not necessarily to fix in this particular PR, more a general thought.)